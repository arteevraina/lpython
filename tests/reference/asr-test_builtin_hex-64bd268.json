{
    "basename": "asr-test_builtin_hex-64bd268",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_builtin_hex.py",
    "infile_hash": "e639f0251477f50376536d317630e3c026354859576a5f1b7b10bd7d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_builtin_hex-64bd268.stderr",
    "stderr_hash": "22b2ae7d0d34d2406125547149511ab5990a5f2163c186c48c5c4f05",
    "returncode": 2
}