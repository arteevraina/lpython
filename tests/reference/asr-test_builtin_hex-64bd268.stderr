warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/../integration_tests/test_builtin_hex.py:6:12
  |
6 |     assert hex(i) == "0x22"
  |            ^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
